### ISSUE TITLE
I get module 'video_player' not found when building on ios

### ISSUE THREAD

**Issue by adolfokrah** [flutter/flutter#91568]
This is my pubspec.yml
```
dependencies:
  flutter:
    sdk: flutter

  cupertino_icons: ^1.0.2
  video_player: ^2.2.5
  get: ^4.3.8
  chewie: ^1.2.2

____________________________________________________________________

Xcode's output:
↳
    /Users/macbook/Desktop/flutterprojects/afri_pro/ios/Runner/GeneratedPluginRegistrant.m:12:9: fatal error: module 'video_player' not found
    @import video_player;
     ~~~~~~~^~~~~~~~~~~~
    1 error generated.
    note: Using new build system
    note: Planning
    note: Build preparation complete
    note: Building targets in parallel

Could not build the application for the simulator.
Error launching application on iPhone 13 Pro Max.
```

**Comment by danagbemava-nc** [comment]
Without additional information, we are unfortunately not sure how to resolve this issue. 
We are therefore reluctantly going to close this bug for now. 
Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!  
Thanks for your contribution.

Could everyone who still has this problem please file a new issue with the exact description of what happens, logs, and the output of `flutter doctor -v`.
All system setups can be slightly different, so it's always better to open new issues and reference related issues.
